tryalgo/dist_grid.py

```diff
@@ -11,29 +14,31 @@ def dist_grid(grid, source, target=None):
     """Distances in a grid by BFS
 
     :param grid: matrix with 4-neighborhood
-    :param (int,int) source: pair of row, column indices
+    :param (int,int) source: pair of row, column ind_ices
     :param (int,int) target: exploration stops if target is reached
     :complexity: linear in grid size
     """
     rows = len(grid)
     cols = len(grid[0])
-    dirs = [(0, +1, '>'), (0, -1, '<'), (+1, 0, 'v'), (-1, 0, '^')]
     i, j = source
     grid[i][j] = 's'
-    Q = deque()
-    Q.append(source)
-    while Q:
-        i1, j1 = Q.popleft()
-        for di, dj, symbol in dirs:   # explore all directions
-            i2 = i1 + di
-            j2 = j1 + dj
-            if not (0 <= i2 and i2 < rows and 0 <= j2 and j2 < cols):
-                continue              # reached the bounds of the grid
-            if grid[i2][j2] != ' ':   # inaccessible or already visited
+    q_ueue = deque()
+    q_ueue.append(source)
+    while q_ueue:
+        i_1, j_1 = q_ueue.popleft()
+        for d_i, d_j, symbol in [(0, +1, '>'),
+                                 (0, -1, '<'),
+                                 (+1, 0, 'v'),
+                                 (-1, 0, '^')]:   # explore all d_irections
+            i_2 = i_1 + d_i
+            j_2 = j_1 + d_j
+            if not (0 <= i_2 < rows and 0 <= j_2 < cols):
+                continue                # reached the bounds of the grid
+            if grid[i_2][j_2] != ' ':   # inaccessible or already visited
                 continue
-            grid[i2][j2] = symbol     # mark visit
-            if (i2, j2) == target:
-                grid[i2][j2] = 't'    # goal is reached
+            grid[i_2][j_2] = symbol     # mark visit
```

Comment:
Did this disturb pylint or is it just you? Cos d_i and i_1 do not have same meaning.

Reply:
pylint was complaining about one-character variable naming (C0103) but we can leave one-character variable naming.